views/js/runner/helpers/messages.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
+ */
+define([
+    'jquery',
+    'lodash',
+    'i18n',
+    'taoQtiTest/runner/helpers/map'
+], function ($, _, __, mapHelper) {
+    'use strict';
+
+    /**
+     * Tells is the current item has been answered or not
+     * The item is considered answered when at least one response has been set to not empty {base : null}
+     *
+     * @returns {Boolean}
+     */
+    function isCurrentItemAnswered(runner) {
+        var answered = false;
+        _.each(runner.itemRunner && runner.itemRunner.getState(), function (state) {
+            var response = state && state.response;
+            if (_.isObject(response)) {
+                answered = _.isObject(response.base);
```

Comment:
This is what we talk it could be also `response.list` or `response.record` according to http://www.imsglobal.org/assessment/pciv1p0cf/imsPCIv1p0cf.html#_Toc353965343